### PR TITLE
Created taxonomy structure #72

### PR DESCRIPTION
**Pull request by anpolimus**
#72 pull request

### CODE REVIEW

**Comment by anpolimus** [comment]
@podarok, need review

File: docroot/profiles/ppdorg/ppdorg.install [hunk 1]
```diff
@@ -40,3 +40,7 @@ function ppdorg_install() {
   user_role_grant_permissions(DRUPAL_ANONYMOUS_RID, array('access content'));
   user_role_grant_permissions(DRUPAL_AUTHENTICATED_RID, array('access content'));
 }
+
+/**
+ * Enable core taxonomy module().
```
Comment (podarok) [inline docroot/profiles/ppdorg/ppdorg.install]: we don't need this comment, right?

**Comment by podarok** [comment]
@anpolimus please add steps for review

**Comment by anpolimus** [comment]
1. taxonomy vocab user list should appear.
2. request query key value field should appear at term add page.
http://screencloud.net/v/CS1H

**Comment by podarok** [comment]
@anpolimus why would we need those request key inputs?

**Comment by anpolimus** [comment]
I will take that parameters in retreiving of user lists from d. org.
For example:
field_user    Ukraine
field_sex      male

Would be transformed to next request: https://www.drupal.org/api-d7/user.json?field_country=Ukraine&field_sex=male

**Pull request merged**